<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoeVSSetting</class>
 <widget class="QWidget" name="MoeVSSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="SettingDeviceID">
       <property name="text">
        <string>SettingDeviceID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="SettingNumThread">
       <property name="text">
        <string>SettingNumThread</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="SettingProvicer">
       <property name="text">
        <string>SettingProvicer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="SettingProvicerComboBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="SettingDeviceIDComboBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="SettingNumThreadComboBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
